.cloudbuild/scripts/cmd/unit-tests/main.go

```diff
@@ -142,16 +144,22 @@ func run() error {
 		}
 	}
 
-	log.Println("Analyzing code changes")
-	ch, err := changes.Analyze(args.workspace, args.targetBranch, args.commitSHA)
-	if err != nil {
-		return trace.Wrap(err, "Failed analyzing code")
-	}
-	log.Printf("Detected changes: %+v", ch)
+	var ch changes.Changes
 
-	if !ch.HasCodeChanges() {
-		log.Println("No code changes detected. Skipping tests.")
-		return nil
+	if args.forceRun {
+		log.Println("Forcing test run")
+	} else {
+		log.Println("Analyzing code changes")
+		ch, err := changes.Analyze(args.workspace, args.targetBranch, args.commitSHA)
```

Comment:
Won't this assignment overshadow the `ch` declared in the outer scope so nothing will ever get run?